drivers/alsa/audio_driver_alsa.cpp

```diff
@@ -171,14 +171,14 @@ void AudioDriverALSA::thread_func(void *p_udata) {
 		ad->start_counting_ticks();
 
 		if (!ad->active) {
-			for (unsigned int i = 0; i < ad->period_size * ad->channels; i++) {
+			for (unsigned long i = 0; i < ad->period_size * ad->channels; i++) {
```

Comment:
I don't know if this is desirable. https://github.com/godotengine/godot/issues/25652